changelog.d/pa-1633.added

```diff
@@ -0,0 +1,18 @@
+Taint: `pattern-propagators` now have optional fields `requires` and `label`, which are used identically to their counterparts
+in `pattern-sources` and `pattern-sinks`.
+
+For instance, we can define:
+```
+pattern-propagators:
+  - pattern: |
+      $TO.foo($FROM)
+    from: $FROM
+    to: $TO
+    requires: A
+    label: B
+```
+to denote a propagator which only propagates from $FROM to $TO if $FROM has taint label A. In addition, it also taints $TO
+with label B.
```

Comment:
Here I understand that the `$TO` will get both label `A` and `B`.

Reply:
hmm, i think i meant this to mean "it additionally (as an action) taints $TO with label B", not "it additionally (as a separate taint) taints $TO with label B". i can rephrase.